### PR TITLE
tools: update pd-control.md

### PR DESCRIPTION
**Pull request by disksing**
See: https://github.com/pingcap/pd/pull/1140

### CODE REVIEW

File: tools/pd-control.md [hunk 1]
```diff
@@ -258,7 +258,7 @@ config set disable-raft-learner true        // 关闭 raft learner 功能
 `disable-remove-down-replica` 用于关闭自动删除 DownReplica 的特性。
 当设置为 true 时，PD 不会自动清理宕机状态的副本。
 
-`disable-make-up-offline-replica` 用于关闭迁移 OfflineReplica 的特性。
+`disable-replace-offline-replica` 用于关闭迁移 OfflineReplica 的特性。
```
Comment (lilin90) [inline tools/pd-control.md]: In line 112, it is "disable-make-up-offline-replica". I think that line also needs to be updated?

**Comment by meyu44** [comment]
@disksing

File: tools/pd-control.md (hunk 1 above)
Comment (disksing) [reply tools/pd-control.md]: Oh I missed it.

**Comment by disksing** [comment]
PTAL @lilin90 @meyu44

**Pull request merged**